iidm/iidm-impl/src/test/java/eu/itesla_project/iidm/network/impl/ReactiveCapabilityCurveTest.java

```diff
@@ -128,4 +128,17 @@ public void invalidMinQ() {
                 .add();
     }
 
+    @Test
+    public void invalidMinQBiggerThenMaxQ() {
```

Comment:
Spelling: invalidMinQGreaterThanMaxQ sounds better?

Reply:
It sounds greater